### ISSUE TITLE
Scrum update 4

### ISSUE THREAD

**Issue by engineereng** [JimmyRowland/team_sporty#20]
1. I reviewed our codebase and helped talk through the difficult process of translating it to GraphQL and Apollo.
2. I found the integration of new technologies very difficult, and I struggled to commit anything meaningful to the project.
3. I will step up and do my best to help finish the assignment by adding functionality to our buttons and sprucing up the page.

i5m1b

**Comment by WilliamJizh** [comment]
1. I made multiple new components and pages in the frontend, as well as the message board functionality on the server-side.
2. We have integrated numerous new technologies into our project during this sprint, I found some difficulties to adopt a completely new framework in a short amount of time. Fortunately, by the end of the sprint, I've became much more comfortable with these techs.
3. During the next sprint, I will keep working on both front and backend and try to fulfill more of our goals.

v2u1b

**Comment by JimmyRowland** [comment]
1. I set up the authentification and apollo boilerplate.
2. It took too much time to set up authentication and apollo boilerplate. 
3. I will try to write more resolvers and queries.

**Comment by gparappilly** [comment]
1. This sprint, I tried to familiarize myself with NextJs, Appolo, GraphQL, and even understand the authentication. I also made the front page and fixed up some of the design for the calendar page.
2. Major issues were the fact that I had to adjust to several new technologies being added. I did find it okay in the end though.
3. This sprint, I plan to hopefully write more queries and setup more of our database models, and finish a lot of the front for certain pages.
